karma.conf.js

```diff
@@ -37,6 +37,8 @@ module.exports = (config) => {
                     },
                 },
             },
+            browserDisconnectTolerance: 2,
+            browserNoActivityTimeout: 60000,
```

Comment:
@cuberoot & @benjamind After trying a handful of the "fixes" that intended to fix the `Disconnected, because no message in 30000 ms.` issue, it seems that only the "give it more times and tried because that's not _supposed_ to happen" approach is the only one that helps in this area... 😞 However, this is that and I've seem much nicer test pass success with this addition.